command/pr_create_test.go

```diff
@@ -407,21 +409,28 @@ func TestPRCreate_survey_defaults_multicommit(t *testing.T) {
 
 func TestPRCreate_survey_defaults_monocommit(t *testing.T) {
 	initBlankContext("", "OWNER/REPO", "feature")
-	http := initFakeHTTP()
-	http.StubRepoResponse("OWNER", "REPO")
-	http.StubResponse(200, bytes.NewBufferString(`
-	{ "data": { "repository": { "forks": { "nodes": [
-	] } } } }
+	http := initMockHTTP()
+	defer http.Verify(t)
```

Comment:
Optional: once the test is done, verify that all response stubs were matched and fail the test otherwise

Reply:
This is a nice idea but I don't mind if you want to do it later

Reply:
Oh, it's done! This demonstrates its use. It's an added level of assertion; I've only indicated that this line is "optional" because the test case would have passed without it too